### DISCUSSION TITLE
report a bug: General Error "404 Not Found nginx" in chrome browser in jetstream laravel official page

### DISCUSSION THREAD

**Discussion by ehsanlesani** [laravel/framework#49050] [Q&A] [answered]
Hello
when I try direct access to all sub pages of jetstream laravel official site in chrome browser, then show "404 Not Found nginx". for example:
https://jetstream.laravel.com/introduction
https://jetstream.laravel.com/stacks/livewire
https://jetstream.laravel.com/features/teams
and all sub page error: "Error 404 Not Found nginx".
this test done for on my system windows 10 and on my Samsung mobile with chrome browser with proxy and without proxy.
this error belong to start kit jetstream in github repository? or this error belong to jetstream laravel official site?
in browser Edge not existing this problem because this browser automatically add .html to request every sub page.
there are 2 solutions if said not problem start kit jetstream:
1) config nginx web server in jetstream laravel official site and add .html to request every sub page.
2) in main page of jetstream laravel official site, add .html end of link addresses to all links on every page.

*** The proof of this error is provided in the appendix:

![Screenshot_20231120_154032_Chrome](https://github.com/laravel/framework/assets/17852752/59174840-89ad-42ef-bad1-66a3e0370c42)
![Screenshot (58)](https://github.com/laravel/framework/assets/17852752/720e1ec0-0b59-434f-aefe-e0ceccd87859)

**Comment by antimech** [comment]
+1
Also it happens when you open a sidebar link in a new tab.

Here is the docs repo: https://github.com/laravel/jetstream-docs/

Forge docs doesn't have such issue: https://forge.laravel.com/docs/introduction.html
Repo: https://github.com/laravel/forge-docs
It also gives 404 when `.html` is not appended, however at least all navigation links end with `.html`: https://github.com/laravel/forge-docs/blob/9051e965b7ea597ccb4377a3126e162d489ab902/.vitepress/config.ts#L70
when in JetStream it is not: https://github.com/laravel/jetstream-docs/blob/6ca644c3daa2bbf0a4862fc1fb2000eb1c0c5b23/.vitepress/config.ts#L78

**Reply by antimech** [reply, marked as answer]
Just started the latest version on my local machine and it works, no issues at all. Seems like a server-side issue.

**Reply by ehsanlesani** [reply]
Thank you very much for your helpful advice.
